src/crypto/openssl/cose_sign.h

```diff
@@ -16,8 +16,12 @@ namespace ccf::crypto
   static constexpr int64_t COSE_PHEADER_KEY_ALG = 1;
   // Standardised: hash of the signing key.
   static constexpr int64_t COSE_PHEADER_KEY_ID = 4;
+  // Standardised: CWT claims map.
+  static constexpr int64_t COSE_PHEADER_KEY_CWT = 15;
   // Standardised: verifiable data structure.
   static constexpr int64_t COSE_PHEADER_KEY_VDS = 395;
+  // Standardised: issued at CWT claim.
+  static const std::string COSE_PHEADER_KEY_IAT = "iat";
```

Comment:
iat is 6, and needs to be NumericDate: https://www.rfc-editor.org/rfc/rfc8392#section-3.1.6